Library/Formula/libsass.rb

```diff
@@ -1,8 +1,9 @@
 class Libsass < Formula
   homepage "https://github.com/sass/libsass"
-  url "https://github.com/sass/libsass/archive/3.1.0.tar.gz"
-  sha1 "858c41405f5ff8b4186c7111e08f29893f4e51a1"
+  url "https://github.com/sass/libsass.git"
+  sha256 "7a57dd96aefa2bfa6975781211cf1fca161ebe8469d0a31cacd57a5e76e7eb5e"
```

Comment:
Need to remove this and the `version` and change the line above to specify the `:tag =>` and `:revision =>`.